components/colorwheel/index.css

```diff
@@ -11,32 +11,33 @@
  * governing permissions and limitations under the License.
  */
 
-/* Windows High Contrast Mode */
-@media (forced-colors: active) {
-	.spectrum-ColorWheel {
-		--highcontrast-colorwheel-border-color-disabled: GrayText;
-		--highcontrast-colorwheel-fill-color-disabled: Canvas;
+.spectrum-ColorWheel {
+	--spectrum-colorwheel-width: var(--spectrum-color-wheel-width);
+	--spectrum-colorwheel-min-width: var(--spectrum-color-wheel-minimum-width);
+	--spectrum-colorwheel-height: var(--spectrum-color-wheel-width);
 
-		forced-color-adjust: none;
-	}
-}
+	/* @TODO: leveraging the rgb token in this case to achieve the desired border color implementation as rgb + opacity are required by the `rgba` function. */
+	--spectrum-colorwheel-border-color-rgb: var(--spectrum-gray-1000-rgb);
+
+	/* @TODO: using explicit value as `color-wheel-border-opacity` is undefined. */
+	--spectrum-colorwheel-border-opacity: 0.1;
+	--spectrum-colorwheel-border-color: rgba(var(--spectrum-colorwheel-border-color-rgb), var(--spectrum-colorwheel-border-opacity));
 
-.spectrum-ColorWheel {
-	--spectrum-colorwheel-width: var(--mod-colorwheel-width, var(--spectrum-color-wheel-width));
-	--spectrum-colorwheel-height: var(--mod-colorwheel-height, var(--spectrum-color-wheel-width));
 	--spectrum-colorwheel-fill-color-disabled: var(--mod-colorwheel-fill-color-disabled, var(--spectrum-disabled-background-color));
 
-	--spectrum-colorwheel-border-width: var(--mod-colorwheel-border-width, var(--spectrum-border-width-100));
+	--spectrum-colorwheel-border-width: var(--spectrum-border-width-100);
 	--spectrum-colorwheel-track-width: var(--mod-colorwheel-track-width, var(--spectrum-color-control-track-width));
```

Comment:
I just noticed that `--spectrum-colorwheel-track-width` and `--spectrum-colorwheel-fill-color-disabled` have their mods defined up here. I thought most of the mods were supposed to be defined like, when we do the CSS property definition, instead of with these custom variable definitions.

Is that something we should change or fix? It might be intentional with these two, but I wanted to bring it up to double check!